src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt

```diff
@@ -39,7 +38,7 @@ private fun syncBlockLocker(corr: Block) {
  */
 object SectionSyncListener : Listener {
 
-    @EventHandler(ignoreCancelled = true, priority = EventPriority.HIGH)
+    @EventHandler(ignoreCancelled = true, priority = EventPriority.MONITOR)
```

Comment:
Why is this changed to monitor?

Reply:
Ah this can be changed back

Reply:
This event does not modify the event and only needs to run after other plugins might have cancelled it